<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSettings</class>
 <widget class="QDialog" name="ChannelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="channelTabName">
    <attribute name="title">
     <string>#channel</string>
    </attribute>
    <widget class="QComboBox" name="colorUserNames">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colorUserNamesLabel">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>20</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Color User Names</string>
     </property>
    </widget>
    <widget class="QComboBox" name="suppressEnterLeaveNotifications">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="suppressEnterLeaveNotificationsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>135</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide Enter/Leave Messages</string>
     </property>
    </widget>
    <widget class="QComboBox" name="joinOnConnect">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="joinOnConnectLabel">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>80</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Join On Connect</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChannelSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChannelSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
